mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/NoImplicit.java

```diff
@@ -18,6 +20,7 @@
  *
  * @since 1.3.0
  */
+@MinMuleVersion("4.3.0")
```

Comment:
Should we just put "4.3" instead? Only mentioning this because on other cases where patch version is 0 I see that it is not specified

Reply:
yes, it makes sense to make it consistent, I will change it.